From 63df256254d8307d8f9a0223148cc4707a2427a5 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <Michal@Hrusecky.net>
Date: Tue, 23 Jan 2018 18:16:51 +0100
Subject: [PATCH] python-ply: Add host build

Might be handy to somebody and we actually need it.

Signed-off-by: Michal Hrusecky <Michal@Hrusecky.net>
---
 lang/python/python-ply/Makefile | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/lang/python/python-ply/Makefile b/lang/python/python-ply/Makefile
index 6970204..4d0ee58 100644
--- a/lang/python/python-ply/Makefile
+++ b/lang/python/python-ply/Makefile
@@ -21,11 +21,14 @@ PKG_LICENSE:=BSD-3-Clause
 PKG_LICENSE_FILES:=README.md
 PKG_MAINTAINER:=Jeffery To <jeffery.to@gmail.com>
 
+include $(INCLUDE_DIR)/host-build.mk
 include $(INCLUDE_DIR)/package.mk
 include ../python-package.mk
+include ../python-host.mk
 include ../python3-package.mk
 
 PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+HOST_UNPACK:=$(HOST_TAR) -C $(HOST_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
 
 define Package/python-ply/Default
   SECTION:=lang
@@ -59,6 +62,15 @@ $(call Package/python-ply/description)
 (Variant for Python3)
 endef
 
+define Host/Compile
+$(call Build/Compile/HostPyMod,,install --prefix="" --root="$(STAGING_DIR_HOSTPKG)")
+endef
+
+define Host/Install
+endef
+
+$(eval $(call HostBuild))
+
 $(eval $(call PyPackage,python-ply))
 $(eval $(call BuildPackage,python-ply))
 $(eval $(call BuildPackage,python-ply-src))
-- 
2.15.1

